src/core/socket.c

```diff
@@ -856,8 +865,9 @@ static int instance_from_socket(int fd, unsigned nr, char **ret) {
                         b = be32toh(remote.in.sin_addr.s_addr);
 
                 if (asprintf(&s,
-                             "%u-%u.%u.%u.%u:%u-%u.%u.%u.%u:%u",
+                             "%u-%" PRIu64 "-%u.%u.%u.%u:%u-%u.%u.%u.%u:%u",
```

Comment:
Won't this break compatibility with existing setups since the naming format will change?

Reply:
we changed that at various times, for example when we included the pidfd id in the name, or when we decided to format ipv4-in-ipv6 addresses in ipv4 format, and so on. 

We never documented this format. I think we should retain the freedom to change this.